nwnltr.c

```diff
@@ -375,6 +446,9 @@ int main(int argc, char *argv[]) {
     else
         load_ltr(cfg.ltrfile, &ltr);
 
+    if (!cfg.build)
```

Comment:
I would add another commandline flag for this. It can be the default, and the flag is --nofix

Reply:
done